### ISSUE TITLE
IDBFS syncing causes error when served from Github.

### ISSUE THREAD

**Issue by rm-code** [TannerRogalsky/love.js#22]
I use github pages to host webdemos for two projects:

- Graphoon: http://rm-code.github.io/Graphoon/
- LOVE-Example-Browser: http://love2d-community.github.io/LOVE-Example-Browser/

It seems like both of them crash at random. Sometimes everything works fine and sometimes it crashes with the following log:
```
Uncaught abort(9) at Error
    at jsStackTrace (http://rm-code.github.io/Graphoon/love.js:1:17162)
    at stackTrace (http://rm-code.github.io/Graphoon/love.js:1:17345)
    at abort (http://rm-code.github.io/Graphoon/love.js:24:13317)
    at Array.LDb (http://rm-code.github.io/Graphoon/love.js:17:239977)
    at rqb (http://rm-code.github.io/Graphoon/love.js:9:475594)
    at pqb (http://rm-code.github.io/Graphoon/love.js:9:475431)
    at oqb (http://rm-code.github.io/Graphoon/love.js:9:474004)
    at nqb (http://rm-code.github.io/Graphoon/love.js:9:473939)
    at __ATINIT__.push.func (http://rm-code.github.io/Graphoon/love.js:1:27935)
    at callRuntimeCallbacks (http://rm-code.github.io/Graphoon/love.js:1:20409)
If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.abort @ love.js:24LDb @ love.js:17rqb @ love.js:9pqb @ love.js:9oqb @ love.js:9nqb @ love.js:9__ATINIT__.push.func @ love.js:1callRuntimeCallbacks @ love.js:1ensureInitRuntime @ love.js:1callMain @ love.js:24(anonymous function) @ love.js:24doCallback @ love.js:1done @ love.js:1IDBFS.reconcile @ love.js:1(anonymous function) @ love.js:1IDBFS.getRemoteSet.IDBFS.getDB.index.openKeyCursor.onsuccess @ love.js:1
(index):99 [post-exception status] All downloads complete.Module.setStatus @ (index):99Module.monitorRunDependencies @ (index):91removeRunDependency @ love.js:1applyMemoryInitializer @ love.js:24xhr_onload @ love.js:1
```

Please let me know if you need more information.

**Comment by TannerRogalsky** [comment]
Apparently the persistent filesystem stuff I put in works on the simple python server but not when served from Github's CDN. This is my first priority when I get a chance to look at it. In the meantime, this is the last known good commit if you wanna check it out and use the files from there: b76a48a8f2db789151c544996d44ad48df53f3df

**Comment by TannerRogalsky** [comment]
Resolved by 55ad3d5.
